navkafka.json

```diff
@@ -1,9 +1,9 @@
 {
   "topics": [
     {
-      "topicName": "privat-syfo-sminfotrygd-manuellBehandling",
+      "topicName": "privat-syfo-sminfotrygd-retry",
```

Comment:
Brukes ikke manuellBehandling-topicen lenger? Hvis den gjør det bør du kanskje heller legge inn en ny entry i stedet for at den gamle blir borte? 

(jeg er klar over at topicen ikke forsvinner, men sånn jeg forstod var disse filene en måte å versjonere oneshoten som brukes for å forvalte topicene, og da bør de jo speile det som faktisk finnes :))

Reply:
Denne topicen, har ikkje vært i bruk i det hele tatt i appen.
Når vi skal sende meldinger til  manuell behandling er er topicen: aapen-syfo-oppgave-produserOppgave, som vi benytter